src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Header from './Header';
+
+const Home = () => (
+  <div>
+    <Header />
```

Comment:
- [ ] [ Optiona] you can actually put the header in your app.js since every page uses it, this will make you not repeat yourself and also make the code base clean